### ISSUE TITLE
The publish jobs of the travis build are broken

### ISSUE THREAD

**Issue by bryce-anderson** [http4s/blaze#115]
Check any recent commit to master: https://travis-ci.org/http4s/blaze/builds/283997160

**Comment by ChristopherDavenport** [comment]
Fix for typelevel scala is on #116 

We still have an error in publishing that is present on 2.10 and probably once we clear the typelevel scala issue on those branches as well

```
java.lang.IllegalStateException: No credential is found for oss.sonatype.org. Prepare ~/.sbt/(sbt_version)/sonatype.sbt file.
```

[Failed Travis for 2.10](https://travis-ci.org/http4s/blaze/jobs/284019464#L2795-L2859) looks to be something to do with the credentials similar to what happened with Rho?

**Comment by bryce-anderson** [comment]
I'm pretty sure this has to do with the shift to sbtRig https://github.com/http4s/blaze/pull/79

**Comment by bryce-anderson** [comment]
Looks like we're getting closer. Now master is attempting to publish signed and thats not going all that well considering no pgp keys are around. https://travis-ci.org/http4s/blaze/jobs/284503157

**Comment by rossabaker** [comment]
I didn't realize this was doing `release with-defaults`.  That's how Verizon projects release, and will try to publish to Sonatype on each successful build.  A more restrained approach is on http4s/http4s, but was a lot of work to set up.  Otherwise, a simple `publish` as opposed to `release with-defaults` might be what's wanted.

What's the goal here?  Full automation, or make it work again?

**Comment by bryce-anderson** [comment]
Considering the pace of blaze releases I was going to switch to a simple publish and do a manual publishSigned for release work. I can get to that later today.

**Comment by bryce-anderson** [comment]
Fixed by commit 32ceeac

https://travis-ci.org/http4s/blaze/builds/284754910
